### PR TITLE
Implementing controlling of the slingshots and cleanups

### PR DESCRIPTION
**Pull request by late-in-autumn**
# Additions
- The players are now only given projectiles when entering battle mode
- All projectiles are now generated by `ProjectileGenerator` component
  - Initial projectiles in the game scene are now removed
- Reloading can be disabled from `ProjectileGenerator` component
- Firing the projectile can be disabled from Projectile component
- A standalone `ControlSlingshotFunctions` component is provided as a wrapper for the above functions, and can be used as a receiver of the Unity messaging system
- `SlingshotLoader` now exposes a new `projectile` field, which allows direct access to the loaded projectile
- All projectile and slingshots carry tags to identify them
- Slingshots reload is now disabled in build mode, and automatically activated when entering battle mode

# Cleanups
- Renamed slingshots to `Slingshot I` and `Slingshot II`
- Renamed the empty game object to `Display and Controls` since it is responsible for the control button and player tags
- Renamed the `tryButton` component to `ControlButton` which I think is more descriptive

### CODE REVIEW

**Comment by late-in-autumn** [comment]
Merge conflict resolved. @asthas please help with verifying that the conflict resolution didn't break your code.

**Comment by asthas** [comment]
LGTM!

**Review by asthas — APPROVED** [review]

File: CodeBenders/Assets/Scripts/ControlButton.cs [hunk 1]
```diff
@@ -56,6 +46,10 @@ void TaskOnClick()
                     enemy.GetComponent<DragDrop>().enabled = false;
 
                 }
+                
+                // enable slingshot reload for player 1
```
Comment (late-in-autumn) [inline CodeBenders/Assets/Scripts/ControlButton.cs:50]: @bzhu36 @bharatht96 you might want to consult these two examples on how to disable slingshot firing for a specific player.

File: CodeBenders/Assets/Scripts/ControlSlingshotFunctions.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
+﻿using UnityEngine;
+
+/// <summary>
```
Comment (late-in-autumn) [inline CodeBenders/Assets/Scripts/ControlSlingshotFunctions.cs:3]: @bzhu36 @bharatht96 also this component shoukl have all the functions you need. You can use this as a message receiver.

**Review by Akxy-01 — APPROVED** [review]

**Pull request merged**